.github/actions/setup/action.yaml

```diff
@@ -23,3 +23,7 @@ runs:
     - name: Create kind cluster
       uses: helm/kind-action@v1.3.0
       if: ${{ inputs.create-kind-cluster == 'true' }}
+
+    - name: Add Dependencies
+      shell: bash
+      run: helm repo add open-telemetry https://open-telemetry.github.io/opentelemetry-helm-charts
```

Comment:
I thought about adding this to the top of `generate-examples` and `check-examples`, but I didn't want affect contributor's local repos.